src/model/chart-model.ts

```diff
@@ -175,6 +175,39 @@ export type VisiblePriceScaleOptions = PriceScaleOptions;
  */
 export type OverlayPriceScaleOptions = Omit<PriceScaleOptions, 'visible' | 'autoScale'>;
 
+/**
+ * Determine how to exit the tracking mode.
+ *
+ * By default, mobile users will long press to deactivate the scroll and have the ability to check values and dates.
+ * Another press is required to activate the scroll, be able to move left/right, zoom, etc.
+ */
+export const enum TrackingExitMode {
+	/**
+	 * Tracking Mode will be deactivated on touch end event.
+	 */
+    OnTouchEnd,
+	/**
+	 * Tracking Mode will be deactivated on the next tap event.
+	 */
+    OnNextTap,
+}
+
+/**
+ * Represent options for the tracking mode's behavior.
+ *
+ * Mobile users will not have the ability to see the values/dates like they do on desktop.
+ * To see it, they should enter the tracking mode. The tracking mode will deactivate the scrolling
+ * and make it possible to check values and dates.
+ */
+export interface TrackingModeOptions {
+	// eslint-disable-next-line tsdoc/syntax
+	/** @inheritdoc TrackingExitMode
+	 *
+	 * @defaultValue `TrackingExitMode.OnNextTap`
```

Comment:
```suggestion
	 * @defaultValue {@link TrackingExitMode.OnNextTap}
```

shouldn't it be like that instead?

Reply:
yes, it's better if it's a link